persistence/sql/sqltest/common.go

```diff
@@ -59,6 +59,150 @@ func startProcess(
 	return startResp.ProcessExecutionId
 }
 
+func terminateProcess(ctx context.Context, ass *assert.Assertions,
+	store persistence.ProcessStore, namespace, processId string) {
+	resp, err := store.StopProcess(ctx, persistence.StopProcessRequest{
+		Namespace:       namespace,
+		ProcessId:       processId,
+		ProcessStopType: xdbapi.TERMINATE,
+	})
+
+	ass.Nil(err)
+	ass.False(resp.NotExists)
+}
+
+func startProcessWithAllowIfPreviousExitAbnormally(ctx context.Context, ass *assert.Assertions,
```

Comment:
It seems like we are missing a test case for the default policy. Maybe add it into the test cases like `SQLProcessIdReusePolicyTerminateIfRunning` to see if it return error with the default policy